test/onnx/onnx_test_common.py

```diff
@@ -461,6 +461,9 @@ def _compare_pytorch_onnx_with_ort(
 # The max onnx opset version to test for
 MAX_ONNX_OPSET_VERSION = _constants.ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET
 TESTED_OPSETS = range(MIN_ONNX_OPSET_VERSION, MAX_ONNX_OPSET_VERSION + 1)
+ONNXRT_SUPPORTED_OPSETS = range(
+    MIN_ONNX_OPSET_VERSION, 19
```

Comment:
Maybe we should create an empty `https://github.com/pytorch/pytorch/blob/main/torch/onnx/symbolic_opset19.py` because we are making references to opset 19 in the code? Also update `torch/onnx/__init__.py`

https://github.com/pytorch/pytorch/pull/84594 is an example PR for how to add a new opset to the exporter. most of it is already done

Reply:
Will do it separately as needed. Currently we don't have any opset19 ops used.